state-sync/state-sync-v2/state-sync-driver/src/tests/driver.rs

```diff
@@ -121,23 +239,31 @@ fn create_driver_for_tests(
     let genesis_txn = Transaction::GenesisTransaction(WriteSetPayload::Direct(genesis));
     bootstrap_genesis::<AptosVM>(&db_rw, &genesis_txn).unwrap();
 
-    // Create the event subscription service and notify initial configs
+    // Create the event subscription service and subscribe to events and reconfigurations
     let storage: Arc<dyn DbReader> = db;
     let synced_version = (&*storage).fetch_synced_version().unwrap();
     let mut event_subscription_service = EventSubscriptionService::new(
         ON_CHAIN_CONFIG_REGISTRY,
         Arc::new(RwLock::new(db_rw.clone())),
     );
-    let reconfiguration_subscriber = event_subscription_service
+    let mut reconfiguration_subscriber = event_subscription_service
         .subscribe_to_reconfigurations()
         .unwrap();
+    let event_key_subscriptions =
+        event_key_subscriptions.unwrap_or_else(|| vec![EventKey::random()]);
+    let event_subscriber = event_subscription_service
+        .subscribe_to_events(event_key_subscriptions)
+        .unwrap();
+
+    // Notify subscribers of the initial configs
     event_subscription_service
         .notify_initial_configs(synced_version)
         .unwrap();
+    reconfiguration_subscriber.select_next_some().await;
 
     // Create consensus and mempool notifiers and listeners
     let (consensus_notifier, consensus_listener) =
-        consensus_notifications::new_consensus_notifier_listener_pair(1000);
+        consensus_notifications::new_consensus_notifier_listener_pair(5000);
```

Comment:
Why did this change?

Reply:
Now that the tests are actually testing this, I've updated the value to match that of prod. I don't think we'll hit it, but it would be good to know. I also don't want the tests to flake if the machine is under heavy load.